### PR TITLE
Feat/network x64

### PR DESCRIPTION
**Pull request by adamdoupe**
Added a test case that:

1. Generates a call_shellcode exploit for a network x86_64 binary.
2. The exploit _actually works_ against the target.

### CODE REVIEW

File: tests/test_rex.py [hunk 1]
```diff
@@ -225,6 +229,48 @@ def test_linux_stacksmash_32():
...
+            exploit.arsenal['call_shellcode'].script(exploit_location)
+
+            exploit_result = subprocess.check_output(["python", exploit_location, "127.0.0.1", new_port, "echo hello"])
+            assert b"hello" in exploit_result
+        finally:
+            os.unlink(exploit_location)
+            
+            
+    
+
```
Comment (ltfish) [inline tests/test_rex.py]: Too many blank lines!

**Comment by ltfish** [comment]
LGTM. I'll hit merge once you remove the extra blank lines. Thanks!

**Comment by adamdoupe** [comment]
Extra newlines removed.

Also the damn rr test cases are breaking things, I'll create an issue for that.

**Comment by ltfish** [comment]
> Also the damn rr test cases are breaking things, I'll create an issue for that.

I'll merge in this PR first.

**Pull request merged**